packages/calcite-components/src/components/block/block.tsx

```diff
@@ -350,6 +402,7 @@ export class Block
             class={CSS.content}
             hidden={!open}
             id={regionId}
+            ref={this.setTransitionEl}
```

Comment:
Can you add `// eslint-disable-next-line react/jsx-sort-props` before `ref` ([example](https://github.com/Esri/calcite-design-system/blob/f38167bfeaf92557f4f0f9a437b6d4b35b72fc9c/packages/calcite-components/src/components/flow-item/flow-item.tsx#L295))? It's the last prop here, but we need to make sure that it's always set last to ensure props are in sync when `ref`'s callback is invoked.

I'll submit a PR to add an explainer to all of these.